packages/core/test/pseudo-states.spec.ts

```diff
@@ -626,6 +649,35 @@ describe('pseudo-states', () => {
                 });
             });
 
+            it('should warn when pseudo-class expects params but none was given', () => {
+                const config = {
+                    entry: `/entry.st.css`,
+                    files: {
+                        '/entry.st.css': {
+                            namespace: 'entry',
+                            content: `
+                            .my-class {
+                                -st-states: state1(string);
+                            }
+                            |.my-class:state1 {}
```

Comment:
do you need the `|` (pipe) in this test?

Reply:
Yep, checking where the diagnostic accord.